packages/testing/src/api/index.ts

```diff
@@ -220,6 +220,44 @@ export function loadFirestoreRules(
   });
 }
 
+export type clearFirestoreDataOptions = {
+  projectId: string;
+  databaseId?: string;
+};
+export function clearFirestoreData(
+  options: clearFirestoreDataOptions
+): Promise<void> {
+  if (!options.projectId) {
+    throw new Error('projectId not specified');
+  }
+
+  const optionsWithDefaults = Object.assign({}, options, {databaseId: '(default)'})
+
+  let client = new EMULATOR.FirestoreEmulator(
```

Comment:
we should think about caching these stubs in the future

Reply:
I agree!